assets/src/components/activities/adaptive/components/common/PartComponent.tsx

```diff
@@ -35,7 +35,42 @@ const PartComponent: React.FC<AuthorProps | DeliveryProps> = (props) => {
   const [componentStyle, setComponentStyle] = useState<CSSProperties>(initialStyles);
 
   const [customCssClass, setCustomCssClass] = useState<string>(props.model.customCssClass || '');
+  const handleiFrameStylingChanges = (currentStateSnapshot: Record<string, unknown>) => {
+    //janus-capi-iframe is the only component that allows a user to change it's position and some other style attributes
+    if (props.type === 'janus-capi-iframe') {
```

Comment:
I don't think we need to filter this out if some other component wants to use it, though I am regretting the `IFRAME` prefix now, but there is no reason that I can think of that changing *any* component this way wouldn't be OK